truffle.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
       host: "localhost",
       port: 8545,
       network_id: "*",
-      gas: 4000000,
+      gas: 4000000
```

Comment:
modern js style guides recommend having comma at the end of every line in object definition.